File: truss-compiler/src/main/java/truss/compiler/symbols/ConversionOperatorSymbol.java
package truss.compiler.symbols;

public class ConversionOperatorSymbol extends OperatorSymbolBase {
    @Override
    public SymbolKind getKind() {
        return SymbolKind.CONVERSION_OPERATOR;
    }

    @Override
    public void accept(SymbolVisitor visitor) {
        if (!visitor.isDone()) {
            visitor.visitConversionOperator(this);
        }
    }
}

File: Truss.Compiler/Symbols/ConversionOperatorSymbol.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Truss.Compiler.Symbols {
    public class ConversionOperatorSymbol : OperatorSymbolBase {
        public override SymbolKind Kind {
            get { return SymbolKind.ConversionOperator; }
        }

        public override void Accept(ISymbolVisitor visitor) {
            if (!visitor.Done) {
                visitor.VisitConversionOperator(this);
            }
        }
    }
}
